app/decorators/islet_decorator.rb

```diff
@@ -12,5 +12,11 @@ def grouped_by_sites_options_for_select
         [site.name, islets]
       end
     end
+
+    def cooling_modes_for_options
```

Comment:
I just notice that we seems to use two way of naming here `_options_for_select` and `_for_options`